wrapper/cpp/main.go

```diff
@@ -206,7 +206,7 @@ func KeyvaultKeyCreateOrUpdateCV(serverName *C.char, groupName *C.char, keyvault
 
 	_, err = keyClient.CreateOrUpdate(ctx, C.GoString(groupName), C.GoString(keyvaultName), C.GoString(keyName), kvConfig)
 	if err != nil {
-		telemetry.EmitWrapperTelemetry("KeyvaultKeyCreateOrUpdateCV", C.GoString(correlationVector), err.Error(), "keyClient.CreateOrUpdate", C.GoString(serverName))
+		telemetry.EmitWrapperTelemetry("KeyvaultKeyCreateOrUpdateCV", C.GoString(cv), err.Error(), "keyClient.CreateOrUpdate", C.GoString(serverName))

```

Comment:
>cv [](http://example.com/codeflow?start=75&length=2)

Since this is parameter name change, the old header should still be compatible right?

Reply:
Once this is in make sure to update the header in igvmagent if you havent already.